.github/workflows/icu_ci.yml

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#
+# GitHub Action configuration script for ICU continuous integration tasks.
+
+name: ICU CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '*'
+
+jobs:
+
+  # ICU4J build and unit test
+  icu4j-build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout and setup
+        uses: actions/checkout@v2
+        with:
+          java-version: 1.11
```

Comment:
What does this line do? I don't think `java-version` is a parameter to the `actions/checkout` action (or if it is, I can't easily find the doc for it), but it wouldn't make sense since conceptually, a `git checkout` and the version of Java you use are orthogonal.

The [`actions/java-setup`](https://github.com/actions/setup-java) command seems to be the way to configure your Java version, and because it is in https://github.com/actions, it means that it is maintained by GH itself.

Reply:
It sets environment:
env:
    JAVA_HOME_11.0.9_x64: /opt/hostedtoolcache/jdk/11.0.9/x64
    JAVA_HOME: /opt/hostedtoolcache/jdk/11.0.9/x64
    JAVA_HOME_11_0_9_X64: /opt/hostedtoolcache/jdk/11.0.9/x64